bin/toggle_read_only

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env escript
+%% -*- erlang -*-
+%%!
+
+%%% - Toggle read-only mode in the local Logplex
+
+
+main([ReadOnly]) when ReadOnly =:= "on" ->
```

Comment:
Good point! This was repurposed from replace_shards. Will make a change.

On July 23, 2015 1:11:00 PM PDT, Alex Arnell <notifications@github.com> wrote:
>> @@ -0,0 +1,74 @@
>> +#!/usr/bin/env escript
>> +%% -*- erlang -*-
>> +%%!
>> +
>> +%%% - Toggle read-only mode in the local Logplex
>> +
>> +
>> +main([ReadOnly]) when ReadOnly =:= "on" ->
>
>It's not necessary to do this pattern match in the guard clause
>```erlang
>main(["on"]) ->
>   ...;
>main(["off"]) ->
>  ....;
>main(_) ->
>  usage().
>```
>would sufice
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/heroku/logplex/pull/137/files#r35365579

-- 
Sent from a piece of glass, in space.